README.md

```diff
@@ -17,149 +17,264 @@ Dead simple fluent API. Customizable. Reusable.
-eight letters between **v** and **n** in the _"validation"_ word. This is the
-same pattern we are used to seeing in _i18n_, _a11y_, _l10n_ ...
+Mix **rules** and **modifiers** together to create complex validations with
+great ease and fluency:
 
-The `v8n` is a validation library which provides you an easy and incredibly
-fluent way to build and run validations. With this, you can construct validation
-strategies as easy as you'd do in the English language.
+```javascript
+v8n()
+  .some.vowel()
+  .some.consonant()
+  .not.every.lowercase()
+  .not.every.uppercase()
+  .not.some.number()
+  .test("Foo5"); // false - no numbers (.not.some.number), please!
+```
 
-The main goal of this library is to be used to validate any kind of data with
-any validation type. There are a lot of useful built-in-rules for you to use,
-and you also can build (and share) your own.
+So fluent that it looks like English:
 
-By mixing rules and modifiers, you can build a ton of different validation
-strategies, using its incredible fluent API.
+```javascript
+v8n()
+  .some.not.uppercase() // expects that some character is not uppercase
+  .test("Hello"); // true
 
-The `v8n` is not intended to be used in a specific application scope, like an
-input field or data model validation.
+v8n()
+  .not.some.uppercase() // expects that none character is uppercase
+  .test("Hello"); // false
+```
 
-Actually, it's designed to be used with any scope, and to aid reusability of
-validation strategies between scopes. So, you can define your validation and use
-it in your input field, in your pre-request logic, in your server-side model,
-whatever. Pretty cool, huh?
+Notice how we made very different validation strategies just by changing the
+order of the modifiers. It's so intuitive that seems to be impossible, but this
+is v8n.
 
-## Features
+### Customizable
 
-- Incredible fluent and chainable API;
-- Useful standard validation rules;
-- Custom validations rules;
-- Asynchronous validation;
-- Reusability;
-- Validation composition;
+Create your own **custom validation rules** in a very intuitive way:
 
-## Installation
+```javascript
+function foo() {
+  return value => value === "bar";
+}
 
-```shell
-# Using npm
-npm install v8n
+v8n.extend({ foo });
+```
 
-# or yarn
-yarn add v8n
+v8n will treat them as built-in ones:
+
+```javascript
+v8n()
+  .string()
+  .foo()
+  .test("bar"); // true
 ```
 
-### Or using a `script` tag:
+### Reusable
 
-```html
-<!-- From unpkg -->
-<script src="https://unpkg.com/v8n/dist/v8n.min.js"></script>
+Export validations just like you're used to do with your JavaScript modules:
 
-<!-- or from jsdelivr -->
-<script src="https://cdn.jsdelivr.net/npm/v8n/dist/v8n.min.js"></script>
+_myValidation.js_
+
+```javascript
+import v8n from "v8n";
+
+export default v8n()
+  .number()
+  .between(50, 100)
+  .not.even();
 ```
 
-## Usage
+and use them anywhere you want:
+
+```javascript
+import myValidation from "../myValidation";
```

Comment:
Use a better name, maybe specialNumber. myValidation is too general for this I feel.